azure-iot-hub/azure/iot/hub/constant.py

```diff
@@ -0,0 +1,10 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""This module defines constants for use across the azure-iot-device package
+"""
+
+VERSION = "2.0.0-preview.10"
+IOTHUB_API_VERSION = "2019-07-01-preview"
```

Comment:
Is this actually used anywhere? Doesn't the service.json define this for us?

Reply:
Correct

---
In reply to: [322886632](https://github.com/Azure/azure-iot-sdk-python/pull/300#discussion_r322886632) [](ancestors = 322886632)